src/compiler/emitter.ts

```diff
@@ -4131,7 +4131,7 @@ export function createPrinter(printerOptions: PrinterOptions = {}, handlers: Pri
         writeSpace();
         nextPos = emitTokenWithComment(SyntaxKind.AsKeyword, nextPos, writeKeyword, node);
         writeSpace();
-        nextPos = emitTokenWithComment(SyntaxKind.NamespaceKeyword, nextPos, writeKeyword, node);

```

Comment:
Here, I would prefer to leave this code and ignore it. If this code changes, we may not realize we didn't update the pos.